pkg/security/probe/eventstream/reorderer/perfmap.go

```diff
@@ -43,20 +44,21 @@ func (m *OrderedPerfMap) Init(mgr *manager.Manager, config *config.Config) error
 	}
 
 	m.perfMap.PerfMapOptions = manager.PerfMapOptions{
-		RecordHandler: m.reOrderer.HandleEvent,
-		LostHandler:   m.handleLostEvents,
-		RecordGetter:  m.recordPool.Get,
+		RecordHandler:    m.reOrderer.HandleEvent,
+		LostHandler:      m.handleLostEvents,
+		RecordGetter:     m.recordPool.Get,
+		TelemetryEnabled: true,
```

Comment:
What is the overhead of telemetry ? Especially when ebpf-check is disabled (in customers envs)

Reply:
This is unrelated to the ebpf check, but is normal prometheus internal telemetry.

Reply:
I did some performance tests and did not see a change to the CPU or memory behavior.

Reply:
I could try and passthrough the setting of whether internal telemetry is enabled to the ebpf-manager